common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -861,6 +862,15 @@ def student_view_data(self, context=None):
                     "file_size": 0,  # File size is unknown for fallback URLs
                 }
 
+        # Include youtube link if there is no encoding for mobile- ie only a fallback URL or no encodings at all
+        # We are including a fallback URL for older versions of the mobile app that don't handle Youtube urls
+        if self.youtube_id_1_0 and "mobile_low" not in encoded_videos:
+            youtube_1_0_url = "https://www.youtube.com/watch?v={}".format(self.youtube_id_1_0)
```

Comment:
I refactored a bit of that method to ensure we always construct youtube url's the same way. Some of the `get_youtube_link` method isn't relevant to our case because it only applies when we have an edx video id.